README.md

```diff
@@ -1 +1,58 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> 
+
+![screenshot](./)
+
+
+## Built With
+
+- Webpack
+- JavaScript (ES6)
+- CSS
+- HTML
+
+## Live Demo
+[Click here to view](https://cwaku.github.io/Leaderboard/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+some knowledge on `npm` webpack and ES6
+
+### Setup
+
+- Clone repository
+- run `npm start`
```

Comment:
- [ ] After cloning a user needs to also change directory using the `cd` command. Kindly add this command.
- [ ] Technically, a user cannot successfully run `npm start` without first installing the packages using `npm install`. Kindly add it as a necessary setup command